msu/utilities/math.nut

```diff
@@ -5,8 +5,8 @@
 		local count = 0;
 		while (_num > 1) 
 		{
-		    _num = _num >> 1;
-		    count++;
+			_num = _num >> 1;
```

Comment:
Why double tab?

Reply:
wdym double tab? it's aligned properly, was just spaces before instead of tabs